3_javascript/javascript1/week2/javascript.js

```diff
@@ -0,0 +1,146 @@
+//Welcome to my second experience in JAVASCRIPT :)
+
+/* console.log("*******     Flight booking fullname function    *******");
```

Comment:
This comment is not explaining anything, you do not need it.

Reply:
done!